### PR TITLE
Add rocky-container-stackhpc element

### PR DESCRIPTION
**Pull request by mnasiadka**
Uses cloud-init with NM support from c9s and explicitly sets network renderer and activator to network-manager.

### CODE REVIEW

**Comment by mnasiadka** [comment]
Once that merges in, I'll create a PR in SKC with built and promoted image.

**Comment by mnasiadka** [comment]
Building in https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6534760289

File: elements/rocky-container-stackhpc/containerfiles/9-stackhpc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM docker.io/rockylinux/rockylinux:9
+
+RUN dnf group install -y 'Minimal Install' --allowerasing && \
+    dnf install -y findutils util-linux \
+    https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/cloud-init-23.1.1-12.el9.noarch.rpm
```
Comment (markgoddard) [inline elements/rocky-container-stackhpc/containerfiles/9-stackhpc:7]: Do we know how long CS9 keeps packages in their repos for?

**Review by markgoddard — CHANGES_REQUESTED** [review]
This could use a README to show how to use it.

Also please add to the element list in the main repo README.

File: elements/rocky-container-stackhpc/containerfiles/9-stackhpc
Comment (markgoddard) [inline elements/rocky-container-stackhpc/containerfiles/9-stackhpc:1]: Might be nice to comment that this is based on https://github.com/openstack/diskimage-builder/blob/master/diskimage_builder/elements/rocky-container/containerfiles/9, both for attribution and to keep it in sync.

File: elements/rocky-container-stackhpc/containerfiles/9-stackhpc (hunk 1 above)
Comment (mnasiadka) [reply elements/rocky-container-stackhpc/containerfiles/9-stackhpc:7]: They are keeping 5 versions of each package, hopefully the updated version lands in RHEL 9.3 and we can stop doing this.

File: elements/rocky-container-stackhpc/containerfiles/9-stackhpc
Comment (mnasiadka) [reply elements/rocky-container-stackhpc/containerfiles/9-stackhpc:1]: Done.

**Comment by mnasiadka** [comment]
> This could use a README to show how to use it.
> 
> Also please add to the element list in the main repo README.

Done.

File: elements/ofed-rpm/README.rst
Comment (markgoddard) [inline elements/ofed-rpm/README.rst:1]: Did you mean to add this?

File: elements/ofed-rpm/install.d/50-ofed-rpm
Comment (markgoddard) [inline elements/ofed-rpm/install.d/50-ofed-rpm:1]: and this

File: elements/ofed-rpm/README.rst
Comment (mnasiadka) [reply elements/ofed-rpm/README.rst:1]: one git add -A too far ;-)

**Review by markgoddard — APPROVED** [review]

**Pull request merged**